lib/api/devicestatus/index.js

```diff
@@ -24,6 +24,9 @@ function configure (app, wares, ctx) {
       q.count = 10;
     }
     ctx.devicestatus.list(q, function (err, results) {
+      results.sort(function(a,b){
+        return new Date(b.created_at) - new Date(a.created_at);
```

Comment:
I think this will override the sort, by default we already sort by `created_at`, but that can be changed with a sort param

https://github.com/nightscout/cgm-remote-monitor/blob/dev/lib/server/devicestatus.js#L44

Reply:
I really don't think we should be sorting the data after it's returned from mongo, the sort should be done in mongo if there's bad data, that needs to be fixed or removed from the db

Reply:
The date change causes the sorting to not work for queries that get mixed data. Wondering if we should implement a data migration that fixes all the data in existing databases so all the dates would in in the UTC format (along with the utcOffset field).

Reply:
but thats bad old data, this will be a performance hit for everyone that has good data, maybe it could be an option, but should still be able to handle the other sorts

Reply:
Yup, so sounds like a data migration on upgrade is the right way to go

Reply:
an admin tool instead of something automatic might be safer, would give people a chance to make a backup, see if they have data that needs to be migrated, etc